client/load_balancer.go

```diff
@@ -21,8 +22,15 @@ func (c *APIClient) CreateLoadBalancer(req models.CreateLoadBalancerRequest) (st
 
 func (c *APIClient) DeleteLoadBalancer(loadBalancerID string) error {
 	path := fmt.Sprintf("/loadbalancer/%s", loadBalancerID)
-	if err := c.client.Delete(path, nil, nil); err != nil {
-		return err
+	// Retry if Timeout
+	for i := 0; i < 2; i++ {
```

Comment:
Open to suggestion here too. I'm not sure if retry should be a constant here or not.